insights/parsers/rhn_logs.py

```diff
@@ -171,3 +171,32 @@ class SearchDaemonLog(LogFileOutput):
         ['STATUS | wrapper  | 2013/01/29 17:04:25 | TERM trapped.  Shutting down.']
     """
     time_format = '%Y/%m/%d %H:%M:%S'
+
+
+@parser('rhn_server_satellite.log')
+class SatelliteServerLog(LogFileOutput):
+    """
+    Class for parsing the ``var/log/rhn/rhn_server_satellite.log`` file
+
+    Sample log contents::
```

Comment:
Please put it in proper format, need to add one blank line after "Sample log contents", please check the html docs before adding blank line, after adding blank line do "make html_debug", check same html doc.